### ISSUE TITLE
installation of nysbc-3DFSC fails

### ISSUE THREAD

**Issue by n1kt0** [I2PC/scipion#1849]
**Describe the bug**
Issuing the command `./scipion install nysbc-3DFSC` fails with the following error on a  freshly installed CentOS 7.5 system.  

**To Reproduce**
1. `git checkout devel`
2. `./scipion install nysbc-3DFSC`

**Expected behavior**
nysbc-3DFSC will be installed.

**Desktop (please complete the following information):**
 - OS: CentOS 7.5
 - Python: miniconda 3.6.7

**Additional context**
```
Building nysbc-3DFSC-2.5 ...
  Skipping command: wget -nv -c -O /home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5.tgz.part http://scipion.cnb.csic.es/downloads/scipion/software/em/nysbc-3DFSC_2.5.tgz                                           
mv -v /home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5.tgz.part /home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5.tgz                                                                          
  All targets exist.
  Skipping command: tar -xzf nysbc-3DFSC_2.5.tgz
  All targets exist.
cd /home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5
conda env create -f environment.yml && touch IS_INSTALLED
Solving environment: done
Preparing transaction: done
Verifying transaction: - 
SafetyError: The package for qt located at /home/nibalysc/Programs/miniconda3/pkgs/qt-5.6.2-4
appears to be corrupted. The path 'bin/qt.conf'
has a sha256 mismatch.
  reported sha256: de038e40fef1776fcde7982b1c387f040162d4451559f3f233883ab43a2b2efa
  actual sha256: 185aae74d18e43a93841f747aced5018ef96393282579be9982e63ccf6b288b1

ClobberError: This transaction has incompatible packages due to a shared path.
  packages: defaults::libgcc-5.2.0-0, defaults::libgfortran-3.0.0-1
  path: 'lib/libgfortran.so.3'


ClobberError: This transaction has incompatible packages due to a shared path.
  packages: defaults::libgcc-5.2.0-0, defaults::libgfortran-3.0.0-1
  path: 'lib/libgfortran.so.3.0.0'




done
Executing transaction: done
Traceback (most recent call last):
  File "/home/nibalysc/Programs/miniconda3/envs/fsc/bin/pip", line 7, in <module>
    from pip import main
ImportError: cannot import name 'main'

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/site-packages/conda/exceptions.py", line 819, in __call__
        return func(*args, **kwargs)
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/site-packages/conda_env/cli/main.py", line 74, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/site-packages/conda_env/cli/main_create.py", line 108, in execute
        installer.install(prefix, pkg_specs, args, env)
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/site-packages/conda_env/installers/pip.py", line 40, in _pip_install_via_requirements
        args, pip_version = pip_args(prefix)
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/site-packages/conda_env/pip_util.py", line 33, in pip_args
        pip_version = subprocess.check_output(ret + ['-V']).decode('utf-8').split()[1]
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/subprocess.py", line 336, in check_output
        **kwargs).stdout
      File "/home/nibalysc/Programs/miniconda3/lib/python3.6/subprocess.py", line 418, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '['/home/nibalysc/Programs/miniconda3/envs/fsc/bin/python', '/home/nibalysc/Programs/miniconda3/envs/fsc/bin/pip', '-V']' returned non-zero exit status 1.

`$ /home/nibalysc/Programs/miniconda3/bin/conda-env create -f environment.yml`

  environment variables:
                 CIO_TEST=<not set>
                CLASSPATH=.:/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/jre/lib
                          :/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/lib:/usr
                          /lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/lib/tools.jar
  CONDA_AUTO_UPDATE_CONDA=false
               CONDA_ROOT=/home/nibalysc/Programs/miniconda3
              JNI_CPPPATH=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/include:/
                          usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/include/li
                          nux
          LD_LIBRARY_PATH=/home/nibalysc/Projects/public/git/scipion/software/lib:/usr/lib64/mpi
                          ch-3.2/lib:/usr/local/cuda-8.0/lib64:/usr/lib64/mpich-3.2/lib
                  MANPATH=/usr/share/man/mpich-3.2:/usr/share/man:/usr/local/share/man
               MODULEPATH=/usr/share/Modules/modulefiles:/etc/modulefiles
                     PATH=/home/nibalysc/Projects/public/git/scipion/software/bin:/usr/lib/jvm/j
                          ava-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/bin:/usr/lib64/mpich-3.
                          2/bin:/usr/local/cuda-8.0/bin:/home/nibalysc/Programs/miniconda3/bin:/
                          home/nibalysc/Programs/miniconda3/bin:/usr/lib64/qt-3.3/bin:/usr/lib64
                          /mpich-3.2/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
                          /sbin:/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-1.el7_6.x86_64/bin
                          :/home/nibalysc/.local/bin:/home/nibalysc/bin:/usr/lib/jvm/java-1.8.0-
                          openjdk-1.8.0.191.b12-1.el7_6.x86_64/bin
          PKG_CONFIG_PATH=/usr/lib64/mpich-3.2/lib/pkgconfig
               PYTHONPATH=/home/nibalysc/Projects/public/git/scipion:/home/nibalysc/Projects/pub
                          lic/git/scipion/software/lib/python2.7/site-packages:/home/nibalysc/Pr
                          ojects/public/git/scipion/software/em/xmipp/lib
           QT_PLUGIN_PATH=/usr/lib64/kde4/plugins:/usr/lib/kde4/plugins:/home/nibalysc/.kde/lib6
                          4/kde4/plugins/:/usr/lib64/kde4/plugins/
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
               WINDOWPATH=1

     active environment : None
       user config file : /home/nibalysc/.condarc
 populated config files : /home/nibalysc/.condarc
          conda version : 4.5.11
    conda-build version : not installed
         python version : 3.6.7.final.0
       base environment : /home/nibalysc/Programs/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /home/nibalysc/Programs/miniconda3/pkgs
                          /home/nibalysc/.conda/pkgs
       envs directories : /home/nibalysc/Programs/miniconda3/envs
                          /home/nibalysc/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.11 requests/2.20.1 CPython/3.6.7 Linux/3.10.0-957.1.3.el7.x86_64 centos/7 glibc/2.17
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
Error: target '/home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5/IS_INSTALLED' not built (after running 'conda env create -f environment.yml && touch IS_INSTALLED')
```

**Comment by azazellochg** [comment]
Hi @n1kt0 ,
looks like a conda/pip issue here. 
You could try to remove "corrupted" qt package (`rm -rf /home/nibalysc/Programs/miniconda3/pkgs/qt-5.6.2-4`), reinstall it (`conda install --force qt`), clean up fsc env (`conda env remove --name fsc`) and try again manually:
```
cd /home/nibalysc/Projects/public/git/scipion/software/em/nysbc-3DFSC_2.5
conda env create -f environment.yml && touch IS_INSTALLED
```

**Comment by n1kt0** [comment]
Thank you @azazellochg,
These steps solve it except the warning for qt. It is because my installed qt version is higher than the specified one in the environment.yml file I think. I tried the manual way before but was unsure if there are additional steps executed afterwards.

I tried also to remove all the version strings from the environment.yml and let conda solve the dependencies. That way I managed to install nysbc-3DFSC through scipion install command without any warnings from conda.  

But that way I still have to check if everything works fine.

Best regards,

Nikita
